config/puma.rb

```diff
@@ -42,3 +42,11 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+pidfile "/tmp/puma.pid"
```

Comment:
would it make sense to keep that pid file in Rails's tmp dir?
we also have puma's statefile, i'd save it in the same place ideally. it can be set via `state_path`